build.gradle

```diff
@@ -348,79 +342,101 @@ subprojects {
         from sourceSets.main.allSource
     }
 
-    artifacts { archives javadocJar, sourcesJar }
```

Comment:
Why is the artifacts block no longer used here, but we still use it other places (like compiler)?

Reply:
those are unused and removed.